src/Orchard.Cms.Web/project.json

```diff
@@ -1,24 +1,24 @@
 {
   "webroot": "wwwroot",
   "version": "2.0.0-*",
-  "dependencies": {
-    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0",
-    "System.Linq": "4.3.0",
-    "System.Collections.Specialized": "4.3.0",
-    "Microsoft.AspNetCore.Mvc.Modules.Hosting": "1.0.0-*",
-    "Orchard.Environment.Shell.Data": "2.0.0-*",
-    "Orchard.DisplayManagement": "2.0.0-*",
-    "Orchard.Hosting.Console": "2.0.0-*",
-    "Orchard.Recipes.Abstractions": "2.0.0-*",
-    "Microsoft.Extensions.Logging.Console": "1.1.0",
-    "Microsoft.AspNetCore.Diagnostics": "1.1.0",
-    "Microsoft.AspNetCore.Mvc.ViewFeatures": "1.1.0",
-    "System.Dynamic.Runtime": "4.3.0",
-    "Microsoft.Extensions.Localization.Abstractions": "1.1.0",
-			"Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design": "1.1.0-*",
-			"sqlite": "3.13.0"
-  },
+	"dependencies": {
+		"Microsoft.AspNetCore.Server.Kestrel": "1.1.0",
+		"Microsoft.AspNetCore.Server.IISIntegration": "1.1.0",
+		"System.Linq": "4.3.0",
+		"System.Collections.Specialized": "4.3.0",
+		"Microsoft.AspNetCore.Mvc.Modules.Hosting": "1.0.0-*",
+		"Orchard.Environment.Shell.Data": "2.0.0-*",
+		"Orchard.DisplayManagement": "2.0.0-*",
+		"Orchard.Hosting.Console": "2.0.0-*",
+		"Orchard.Recipes.Abstractions": "2.0.0-*",
+		"Microsoft.Extensions.Logging.Console": "1.1.0",
+		"Microsoft.AspNetCore.Diagnostics": "1.1.0",
+		"Microsoft.AspNetCore.Mvc.ViewFeatures": "1.1.0",
+		"System.Dynamic.Runtime": "4.3.0",
+		"Microsoft.Extensions.Localization.Abstractions": "1.1.0",
+		"Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design": "1.1.0-*",
+		"Microsoft.Data.Sqlite": "1.1.0"
```

Comment:
Yes, we compile then do a part of built and also a part of publish for `Compile only assemblies`, `Specific runtime assemblies` ... But not for `Native assemblies`. As mentioned in the wiki page.

>Native assemblies: Some libraries need to provide native implementations for different runtimes environments. Because modules are not intended to be published themselves, we don't care about native outputs.

Don't remember the details, i don't know if it's doable.